app/components/school-session-attributes-expanded.js

```diff
@@ -0,0 +1,109 @@
+import Ember from 'ember';
+import { task } from 'ember-concurrency';
+
+const { Component, RSVP, inject, computed, isEmpty } = Ember;
+const { service } = inject;
+const { all } = RSVP;
+
+export default Component.extend({
```

Comment:
i see a lot of similarities between this and the corresponding collapsed component. 
if possible, refactor this out, or perhaps extend the collapsed component (from what i can see, only `tagName` would have to be overridden).